packages/shared/src/services/ai/providers/groq-provider.ts

```
@@ -0,0 +1,1155 @@
+    configJson: string
+  ): Promise<DeploymentConfigResult> {
+    if (!request.projectContext?.projectPath) {
+      throw new Error('Project path is required for bash configuration');
+    }
+
+    // Clean up the response - remove any tool-calling syntax or analysis text
+    let cleanedContent = configJson;
+
+    // Remove common tool-calling patterns that might appear at the start
+    cleanedContent = cleanedContent.replace(
+      /^I'll\s+analyze[^]*?(?=<write_file>|<write>|###|```)/i,
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)